htdocs/core/class/extrafields.class.php

```diff
@@ -2681,7 +2681,7 @@ public function showSeparator($key, $object, $colspan = 2, $display_type = 'card
 	 * @param	int			$todefaultifmissing 1=Set value to the default value in database if value is mandatory and missing
 	 * @return	int								1 if array_options set, 0 if no value, -1 if error (field required missing for example)
 	 */
-	public function setOptionalsFromPost($extralabels, &$object, $onlykey = '', $todefaultifmissing = 0)
+	public function setOptionalsFromPost($extralabels, $object, $onlykey = '', $todefaultifmissing = 0)
```

Comment:
Line 2827, the code change a value of $object->array_options to have $object modified after function.
Does this still works if you remove the & here ?

Reply:
yes it still works
<img width="644" height="108" alt="image" src="https://github.com/user-attachments/assets/70173592-1f67-4b95-9289-e00dd8ae47a1" />
changing value from aaa to bbb
<img width="385" height="123" alt="image" src="https://github.com/user-attachments/assets/c6141a76-1abe-47c2-805f-e66331d81076" />